### PR TITLE
in-progress PR

### PR DESCRIPTION
**Pull request by mouseratti**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from setuptools import setup
 
 setup(
-    setup_requires=['pbr'],
+    setup_requires=['pbr', 'pytest-runner', 'wheel'],
```
Comment (Stibbons) [inline setup.py]: add to requirements-dev.txt please

File: guake/__main__.py [hunk 2]
```diff
@@ -23,10 +23,10 @@
 from __future__ import unicode_literals
 
 import logging
+import sys
 
-from guake.logging import setupBasicLogging
-from guake.logging import setupLogging
-
+from guake.logging import setupBasicLogging, setupLogging
```
Comment (Stibbons) [inline guake/__main__.py]: please keep each import on its own line. This make maintainance much simpler

File: guake/test_terminal.py [hunk 3]
```diff
@@ -26,14 +26,9 @@
...
-gi.require_version('Gtk', '3.0')
-gi.require_version('Notify', '0.7')
-gi.require_version('Vte', '2.91')
-
-from gi.repository import GLib
-from gi.repository import Gtk
-from gi.repository import Vte
+from guake import gi
+
+from gi.repository import GLib, Gtk, Vte
```
Comment (Stibbons) [inline guake/test_terminal.py]: I had errors about the version, that's why I had to add the require_version before, how did you manage to get rid of them?

File: guake/test_terminal.py [hunk 4]
```diff
@@ -43,7 +38,7 @@
 def main():
     terminal = Vte.Terminal()
 
-    terminal.spawn_sync(
+    terminal.fork_command_full(
```
Comment (Stibbons) [inline guake/test_terminal.py]: hum, vte 0.38 has replaced fork_command_full by spawn_sync, which version of VTE do you have?

File: guake/test_terminal.py (hunk 3 above)
Comment (mouseratti) [inline guake/test_terminal.py]: see [guake/__init__.py](https://github.com/Guake/guake/pull/880/files#diff-6ca81edf8e502556eebff7aa46fbc5c1)
In my opinion, it`s better to keep this version bindings in a single place.

Comment (Stibbons) [inline guake/test_terminal.py]: if it works without complaining, ok

File: guake/test_terminal.py (hunk 4 above)
Comment (mouseratti) [inline guake/test_terminal.py]: fixed

**Comment by mouseratti** [comment]
@Stibbons please, check it again

**Comment by mouseratti** [comment]
I would also like to move:
-  **guake** directory to **src/guake**
-  **guake/tests** to **src/tests**

What do you think about?

**Comment by Stibbons** [comment]
I merge. Ok you can move these files to the src directory. Dont forget to update setup.cfg

**Comment by Stibbons** [comment]
i'll try to fix the code style and have `./install.py --checks` happy. Please try to have pylint, pep8 and pyflakes happy! I'd like to follow as much as possible the google code style and most of the pep8 recommendations

**Pull request merged**